app/services/meteors.service.ts

```diff
@@ -0,0 +1,47 @@
+import fetchMeteors from "../repositories/meteors.repository.js";
+import { IFormattedMeteor } from "../types/meteors.types.js";
+import { validateDate } from "../utils/utils.js";
+
+interface IQueryResponse {
+  count?: number;
+  wereDangerousMeteors?: boolean;
+}
+
+export default async function getFormattedMeteors(
+  dates: string | string[] | undefined,
+  count: string | undefined,
+  wereDangerousMeteors: string | undefined,
+) {
+  const [startDate, endDate] = validateDate(dates);
```

Comment:
Return it to the controller layer please
The controller is responsible for preparing the data and it must arrive at the service in the form expected by the business logic. I would also like to suggest you rename this function since it does not validate the data